src/components/policy/policy_external/src/sql_pt_representation.cc

```diff
@@ -869,6 +924,37 @@ bool SQLPTRepresentation::SaveRpcs(int64_t group_id,
   return true;
 }
 
+bool SQLPTRepresentation::SaveExternalConsentEntities(
+    const int64_t group_id,
+    const policy_table::DisallowedByExternalConsentEntities& entities,
+    ExternalConsentEntitiesType type) const {
+  utils::dbms::SQLQuery query(db());
+  if (!query.Prepare(sql_pt::kInsertExternalConsentEntity)) {
+    LOG4CXX_WARN(logger_,
+                 "Incorrect insert statement for external consent entities.");
+    return false;
+  }
+
+  const std::string on_off =
```

Comment:
@istoilovgithub on_off is not the best name. probably entities_type_status, external_concents_status.